ui-v2/app/templates/dc/nodes/show/services.hbs

```diff
@@ -1,3 +1,12 @@
+{{#let (selectable-key-values
+    (array "Name:asc" "A to Z")
+    (array "Name:desc" "Z to A")
+    (array "Status:asc" "Unhealthy to Healthy")
+    (array "Status:desc" "Healthy to Unhealthy")
+      selected=sortBy
+  )
+  as |sort|
+}}
```

Comment:
I think you can take this out for the moment also if you aren't sorting yet?

Reply:
Oops. Removed.